getting-started/5_start_ui.md

```diff
@@ -0,0 +1,16 @@
+Back in our original terminal, we can now start the UI. Before we do so, we need to make a small adjustments for this environment.
+
+First, add the below line to `ui/.env`{{open}}
```

Comment:
```suggestion
First, add the below line in the IDE tab to the `ui/.env` file{{open}}
```

Reply:
So _in theory_ clicking on that link should open the IDE in that file. The VsCode integration just doesn't work yet. I've raised that with support.